### ISSUE TITLE
Problems with autoload.php in 4.0.0

### ISSUE THREAD

**Issue by tyranron** [kahlan/kahlan#277]
There are problems with running recent `4.0.0` release.

If I run `kahlan` outside any project (just as binary), it complains about absent `composer.json` file:
```
Warning: file_get_contents(/app/composer.json): failed to open stream: No such file or directory in /usr/src/kahlan-4.0.0/autoload.php on line 21
```

If I run `kahlan` inside project dir (`composer.json` is present), it complains about absent `autoload_files.php` file (if project does not use one):
```
Warning: require(/app/vendor/composer/autoload_files.php): failed to open stream: No such file or directory in /usr/src/kahlan-4.0.0/autoload.php on line 50

Fatal error: require(): Failed opening required '/app/vendor/composer/autoload_files.php' (include_path='.:/usr/local/lib/php') in /usr/src/kahlan-4.0.0/autoload.php on line 50
```

#### How to reproduce:

1. Checkout `dev` branch of [kahlan/docker-image](https://github.com/kahlan/docker-image) repo.

2. Build `4.0.0` Docker image with command:    
    ```bash
    make image DOCKERFILE=4.0/alpine VERSION=alpine
    ```

3. Run simple `kahlan --help` in Docker container:
    ```bash
    docker run --rm kahlan/kahlan:alpine --help
    ```

4. Or run `test/app/` specs in Docker container:
    ```bash
    docker run --rm -v $(pwd)/test/app:/app kahlan/kahlan:alpine
    ```

**Comment by jails** [comment]
Good catch :+1:, fixed in https://github.com/kahlan/kahlan/commit/4cf22224df11b11bd60370ff35cf37daee72bc59

**Comment by tyranron** [comment]
@jails thanks for quick reaction.

While I'm still [facing the issue](https://travis-ci.org/kahlan/docker-image/jobs/284143992) in [Docker tests](https://github.com/kahlan/docker-image/blob/7ac947ecf73937273f580eb3f06fa81c29fdcc46/test/suite.bats) when Kahlan runs outside any project (`composer.json` is not present).
```bash
$ docker run --rm --entrypoint /kahlan-phpdbg kahlan/kahlan:alpine  --help
You need to set up the project dependencies using the following commands: 
curl -s http://getcomposer.org/installer | php
php composer.phar install
```

Can this be fixed? Or this is expected behavior and I just need to update tests?

**Comment by jails** [comment]
Arf yeah, sorry for the mess, should now be fixed in the [4.0.2](https://github.com/kahlan/kahlan/commit/3986051f43da50d35228aa9f6485a8d52d1806e9)
